Source/CPack/cmCPackDebGenerator.cxx

```diff
@@ -110,6 +112,9 @@ int cmCPackDebGenerator::PackageOnePack(std::string initialTopLevel,
     retval = 0;
     }
   // add the generated package to package file names list
+  packageFileName = this->GetOption("CPACK_TOPLEVEL_DIRECTORY");
```

Comment:
`this->ReadListFile` executes `CpackDeb.cmake` after the first assignment. At the second time I just pick up script results.